BUG: support psycopg3 in .to_postgis() (#3167)
geopandas/geopandas: doc/source/getting_started/install.rst

Further, optional dependencies are:

- `pyogrio`_ (optional; faster alternative for fiona, will become the default in GeoPandas 1.0)
- `psycopg`_ (optional; for PostGIS connection)

> - `psycopg2`_ (optional alternative to psycopg; for PostGIS connection)

Reviewer: ```suggestion
- `psycopg2`_ (optional; for PostGIS connection - older version of `psycopg` library)
```
